.yarnrc.yml

```diff
@@ -4,4 +4,4 @@ plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
     spec: "@yarnpkg/plugin-workspace-tools"
 
-yarnPath: .yarn/releases/yarn-berry.cjs
+yarnPath: .yarn/releases/yarn-2.4.1.cjs
```

Comment:
Do we have a corresponding change that we need to make to the CI/CD pipelines for this as well?

Reply:
Nope, CI uses the `yarn` command that's smart enough to find this same file (CI clones the repo).